cli/src/lib/github.js

```diff
@@ -1,13 +1,12 @@
 // @flow
-
 import {Octokit} from '@octokit/rest';
 
-const CLIENT = new Octokit();
-
-if (process.env.GH_TOK) {
-  CLIENT.authenticate({type: 'oauth', token: process.env.GH_TOK});
```

Comment:
This is very old code that isn't actually correct. Updated with new API and reflected in docs